kano_greeter/__init__.py

```diff
@@ -8,3 +8,16 @@
 
 __author__ = 'Kano Computing Ltd.'
 __email__ = 'dev@kano.me'
+
+import os
+import sys
+
+DIR_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+if DIR_PATH != '/usr':
+    sys.path.insert(1, DIR_PATH)
+    LOCALE_PATH = os.path.join(DIR_PATH, 'locale')
+else:
+    LOCALE_PATH = None
+
+import kano_i18n.init
+kano_i18n.init.register_domain('kano-profile', LOCALE_PATH)
```

Comment:
I don't think that this should be `kano-profile` here - it should be an identifier for the greeter translations, no?